pkg/kubelet/kubelet.go

```diff
@@ -1671,6 +1679,13 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 
 		// start syncing lease
 		go kl.nodeLeaseController.Run(context.Background())
+
+		// During node startup, it is possible that the static pod is running but mirror pod was not created because
+		// either the node was not registered fast enough or node informers were not synced. The mirror pods will be
+		// created eventually when the static pods are resynced 1-1.5 mins later. However, this also affects node
+		// readiness latency as we gate node readiness on initial static pods being registered. To improve the node
+		// readiness latency, we want to ensure mirror pods exists as soon as the node is registered.
+		go kl.fastStaticPodsRegistration()
```

Comment:
This function will race on execution since the static pods are not loaded yet. The static pod load, IIRC, happens in syncLoop() down below.

Reply:
don;t call both the same method with this change?

Reply:
With the other changes that have gone in, I think this is ok with the readiness loop and cancellation context that have been added.